xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/VideoFilterShader.cpp

```diff
@@ -77,49 +77,116 @@ BaseVideoFilterShader::BaseVideoFilterShader()
   PixelShader()->SetSource(shaderp);
 }
 
+BaseVideoFilterShaderGLES::BaseVideoFilterShaderGLES()
```

Comment:
the diff between OpenGL and GLES gets too big for keeping both implementations in the same file